.github/workflows/test.yml

```diff
@@ -70,12 +74,14 @@ jobs:
           payload: |
             author: ${{ github.event.sender.login }}
             channel_id: ${{ secrets.SLACK_CHANNEL_ID }}
-            event_url: ${{ env.EVENT_URL}}
+            event_url: ${{ steps.push.outputs.url || steps.pull_request.outputs.url }}
```

Comment:
📣 TIL writing to `$GITHUB_ENV` is not a good practice!

🔗 https://docs.zizmor.sh/audits/#github-env

Some workaround was required to keep the noted step as a single step with various possible inputs, but this continues to match expectations:

- `push`: https://github.com/zimeg/slack-github-action/actions/runs/15056293205/job/42322838794#step:12:8
- `pull_request_target`: https://github.com/zimeg/slack-github-action/actions/runs/15056529213/job/42323524886#step:12:8

Both links above use changes on a fork's `main` branch!